### PR TITLE
Support Win 10 RS4(1803)

### PR DESCRIPTION
**Pull request by puppywang**
1. Using Debian for underlying system.
2. Fix path error on Win 10 RS4 (1803).

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -45,28 +45,35 @@
 
 print('%s[*]%s Probing the Linux subsystem...' % (Fore.GREEN, Fore.RESET))
 
-basedir, lxpath = probe_wsl()
+basedir, lxpath, bashpath = probe_wsl()
+#fix basedir to add LocalState\rootfs
```
Comment (pkulev) [inline install.py]: Missed space after # here and below. All python linters will mark this as an error.

File: install.py [hunk 2]
```diff
@@ -45,28 +45,35 @@
...
+basedir = os.path.join(basedir, 'LocalState')
+rootFsDir = os.path.join(basedir, 'rootfs')
```
Comment (pkulev) [inline install.py]: CamelCase is prohibited in variable names.

**Comment by assapir** [comment]
I tried to use this commit to fix the problem on RS5 insiders, but:
```
PS C:\dev\WSL-Distribution-Switcher> python .\install.py centos
[*] Probing the Linux subsystem...
[!] Unable to find lxrun.exe in the expected locations.
```

**Comment by puppywang** [comment]
@pkulev I update the code style, so it should be ok for a lint.

@meijin007 I will try inside version, maybe lxrun is gone...

**Comment by ghost** [comment]
This mostly works in Windows build 17134.  However, it modifies the distro folder that is the current default, instead of installing into %LOCALAPPDATA%\packages.

**Comment by benzBrake** [comment]
> 
> 
> I tried to use this commit to fix the problem on RS5 insiders, but:
> 
> ```
> PS C:\dev\WSL-Distribution-Switcher> python .\install.py centos
> [*] Probing the Linux subsystem...
> [!] Unable to find lxrun.exe in the expected locations.
> ```
edit `util.py`, remove line 225
```
225-		sys.exit(-1)
```

**Comment by lygstate** [comment]
ping to merge. Win10 RS5

**Pull request merged**